apps/chat/src/store/files/files.reducers.ts

```diff
@@ -164,6 +175,31 @@ export const filesSlice = createSlice({
         ),
       );
       state.filesStatus = UploadStatus.LOADED;
+
+      const idsToReselect = state.chosenEmptyFoldersIds.reduce(
```

Comment:
```suggestion
      const idsToReselect = state.chosenEmptyFoldersIds.reduce<{ folderIds: string[], fileIds: string[] }>(
```

I think it's better to use generics than type assertions